src/applications/accreditation/21a/components/06-background-information-chapter/BackgroundInformationIntro.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+const BackgroundInformationIntro = () => (
+  <>
+    <p>
+      Over the next few pages, we will ask you to provide information about your
+      character, background, and reputations. You will need to provide
+      information about the following information:
+    </p>
+    <ul>
+      <li>Any convictions, imprisonments, sentences to probation or parole</li>
+      <li>Current charges or investigations for any violations of law</li>
+      <li>
+        Suspension, expulsion, resignation or withdrawal from an educational
+        institution
+      </li>
+      <li>
+        Discipline, reprimand, suspension, or termination from a job for
+        unethical or unlawful conduct
+      </li>
+      <li>
+        Any period of service as a representative, agent, or attorney before a
+        state or Federal department or agency, or as a Veterans Service
+        Organization representative
+      </li>
+      <li>
+        Any condition or impairment that could affect your ability to represent
+        claimants in a competent and professional manner
+      </li>
+    </ul>
+    <p>
+      Truthfulness and candor are essential elements of good moral character and
+      reputation relevant to practice before the Department of Veterans Affairs.
+      It is in your best interest; therefore, to provide the Office of the
```

Comment:
Really minor grammar point:
```suggestion
      It is in your best interest, therefore, to provide the Office of the
```